docs/source/installation.rst

```diff
@@ -3,43 +3,119 @@
 Installation
 ============
 
-sktime is available via PyPI using:
+``sktime`` currently supports:
+
+* environments with python version 3.6, 3.7, or 3.8.
+* operating systems Mac OS X, Unix-like OS, Windows 8.1 and higher
```

Comment:
Why not call them Linux? The wheels directly reference linux. 
```suggestion
* operating systems Mac OS X, Linux, Windows 8.1 and higher
```

Reply:
well, I'm not sure - is it really just GNU/Linux distributions and FreeBSD? Also, afaik, FreeBSD is not a GNU/Linux distribution but a Unix-like. Of course it's possible we don't support all unix-likes. But on FreeBSD you would still use manylinux, right?

Reply:
Not sure either, happy to leave it as it is or do something like
```suggestion
* operating systems Mac OS X, Unix-like OS (Linux), Windows 8.1 and higher
```

Reply:
but that would exclude freeBSD

Reply:
Alright I've never heard "unix-like" before and guess that "linux" is much more commonly used but it's a minor point